hummingbot/connector/gateway_EVM_AMM.py

```diff
@@ -645,10 +646,20 @@ async def _http_client(self) -> aiohttp.ClientSession:
             self._shared_client = aiohttp.ClientSession(connector=conn)
         return self._shared_client
 
+    @staticmethod
+    async def get_gateway_connectors(self):
+        return await self._api_request("get", "config", {})
+
+    @staticmethod
+    async def ping_gateway(self):
+        return await self._api_request("get", "", {}, fail_silently = True)
+
+    @staticmethod
     async def _api_request(self,
```

Comment:
ditto.

Also, since you're exposing this to external modules, it should not have the underscore prefix. Underscores are a convention in Python to mean private variables or methods.